libsolidity/formal/CHC.cpp

```diff
@@ -1254,6 +1297,13 @@ vector<smtutil::Expression> CHC::currentStateVariables(ContractDefinition const&
 	return applyMap(SMTEncoder::stateVariablesIncludingInheritedAndPrivate(_contract), [this](auto _var) { return currentValue(*_var); });
 }
 
+smtutil::Expression CHC::currentInitialConstraints(vector<VariableDeclaration const*> const& _vars) const
+{
+	return fold(_vars, smtutil::Expression(true), [this](auto&& _conj, auto _var) {
```

Comment:
The initial value of the `fold` can be passed as a parameter (`conj` in the above case).

Reply:
Sorry, I don't get it. The initial value needs to be `true`